0x000.md

```diff
@@ -0,0 +1,3 @@
+# 0x000 :octocat:
```

Comment:
Right :laughing:. Sorry for not catching that (I was wondering why it was not generating the HTML...)!